src/sonic-host-services-data/templates/common-auth-sonic.j2

```diff
@@ -22,11 +22,16 @@ auth	[success=done new_authtok_reqd=done default=ignore{{ ' auth_err=die' if not
 auth	[success=1 default=ignore]	pam_tacplus.so server={{ last_server.ip }}:{{ last_server.tcp_port }} secret={{ last_server.passkey }} login={{ last_server.auth_type }} timeout={{ last_server.timeout }} {% if last_server.vrf %} vrf={{ last_server.vrf }} {% endif %} {{ 'source_ip=%s' % src_ip if src_ip }} try_first_pass
 
 {% endif %}
-{% elif auth['login'] == 'tacacs+' or auth['login'] == 'tacacs+,local' %}
+{% elif auth['login'] == 'tacacs+' %}
 {% for server in servers %}
```

Comment:
The TACACS sever loop code are same.
2 place changed:

For TACACS+ only scenario, following line removed:
auth [success=1 default=ignore] pam_unix.so nullok try_first_pass

For TACACS+, LOCAL scenario, following line added:
auth [success=done new_authtok_reqd=done default=ignore{{ ' auth_err=die' if not auth['failthrough'] }}] pam_unix.so nullok try_first_pass

I will verify this again

Reply:
The TACACS authorization code be there for long time, and in practice local user can't login when TACACS+ only enabled and remote server reachable.

So, need reproduce and verify this issue.